### PR TITLE
feat: Add capabilities support for BPFLSM

### LINKED ISSUE #1538: feat(enforcer) : Capabilities support
## Feature Request

**Short Description**
Currently, we do not support capabilities with BPFLSM enforcer.  

Task lists 
- [x] Support for enforcement of capabilities rules - ( using  CAPABLE hook )  https://github.com/kubearmor/KubeArmor/pull/1543
- [ ] Implement observability for Linux capabilities in ebpf monitor  (probing cap_capable( ) function to trace capabilities)

### PR DESCRIPTION
**Pull request by Aryan-sharma11**
**Purpose of PR?**:

Fixes #1538 

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
This PR adds  the feature to support enforcement of capabilities rule in BPFLSM.

- We are making use of  `CAPABLE` hook for the purpose. It by default provides us with the capabillity being used by the process.
- Sample[ policy ](https://github.com/kubearmor/KubeArmor/blob/719ae8563da75f877966f7d2cd675ed9187f1100/examples/multiubuntu/security-policies/ksp-ubuntu-1-cap-net-raw-block.yaml#L11) .

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: tests/k8s_env/ksp/ksp_test.go [hunk 1]
```diff
@@ -250,6 +249,9 @@ var _ = Describe("Ksp", func() {
 
 		It("it can block all network traffic on net-raw protocol", func() {
 			//  multiubuntu_test_28, github_test_14
+			if strings.Contains(K8sRuntimeEnforcer(), "bpf") {
+				Skip("Skipping due to policy not supported by bpflsm enforcer")
+			}
```
Comment (daemon1024) [inline tests/k8s_env/ksp/ksp_test.go]: Why is this test skipped

**Review by daemon1024 — DISMISSED** [review]

**Review by Prateeknandle — DISMISSED** [review]
LGTM 👍🏼 
Great work @Aryan-sharma11

**Comment by daemon1024** [comment]
Need to handle edge case where fromsource might be missing

**Review by daemon1024 — APPROVED** [review]

**Review by Prateeknandle — APPROVED** [review]
